client/src/commands.ts

```diff
@@ -116,6 +116,34 @@ function getTemplateTcb(
   };
 }
 
+/**
+ * Command goToComponentWithTemplateFile finds components which reference an external template in
+ * their `templateUrl`s.
+ *
+ * @param ngClient LSP client for the active session
+ */
+function goToComponentWithTemplateFile(ngClient: AngularLanguageClient): Command {
+  return {
+    id: 'angular.goToComponentWithTemplateFile',
+    isTextEditorCommand: true,
+    async execute(textEditor: vscode.TextEditor) {
+      const locations = await ngClient.getComponentsForOpenExternalTemplate(textEditor);
+      if (locations === undefined) {
+        return;
+      }
+
+      vscode.commands.executeCommand(
+          'editor.action.goToLocations',
+          textEditor.document.uri,
+          new vscode.Position(
+              textEditor.selection.start.line, textEditor.selection.start.character),
```

Comment:
Is this referring to the selection in the source document?
I think we can use `textEditor.selection.active`, which is already a `vscode.Position`.
```ts
    /**
     * Represents a text selection in an editor.
     */
    export class Selection extends Range {

        /**
         * The position at which the selection starts.
         * This position might be before or after [active](#Selection.active).
         */
        anchor: Position;

        /**
         * The position of the cursor.
         * This position might be before or after [anchor](#Selection.anchor).
         */
        active: Position;

        // ....
}
```